src/protocols/WaylandProtocol.hpp

```diff
@@ -35,6 +35,6 @@ class IWaylandProtocol {
     IWaylandProtocolDestroyWrapper m_liDisplayDestroy;
 
   private:
-    std::string m_szName;
+    std::string m_szName  = "";
```

Comment:
isnt the default ctor for string exactly this

Reply:
yeah std::string is such a special type it seems. reverted